tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -2785,6 +2856,15 @@ lldp/test_lldp.py::test_lldp_neighbor_post_orchagent_reboot:
 #######################################
 #####           macsec            #####
 #######################################
+
+macsec:
+  skip:
+    reason: "Test is skipped for Mellanox SN5640 platform,dualtor and for  Arista-7060X6-64PE-B-* TH5 SKUs"
+    conditions_logical_operator: "OR"
+    conditions:
+      - "hwsku in ['Arista-7060X6-64PE-B-C448O16','Arista-7060X6-64PE-B-C512S2','Mellanox-SN4700-V64','Mellanox-SN5640-C512S2','Mellanox-SN5640-C448O16']"
```

Comment:
@bachalla please check our discussion.
4700 should not in the hwsku list, it belongs to dualtor.